doc/lua_api.md

```diff
@@ -2368,28 +2348,27 @@ Table of resulting tool uses:
         2   180    60    20     -     -
         3   180    60    20     -     -
 
-**Notes**:
+> **Note**:
+>   * At `crumbly==0`, the node is not diggable.
+>   * At `crumbly==3`, the level difference digging time divider kicks in and makes
+>    easy nodes to be quickly breakable.
+>   * At `level > 2`, the node is not diggable, because it's `level > maxlevel`
 
-* At `crumbly==0`, the node is not diggable.
-* At `crumbly==3`, the level difference digging time divider kicks in and makes
-  easy nodes to be quickly breakable.
-* At `level > 2`, the node is not diggable, because it's `level > maxlevel`
 
 
-
-
-Entity damage mechanism
-=======================
+# Entity Damage Mechanism
 
 Damage calculation:
 
-    damage = 0
-    foreach group in cap.damage_groups:
-        damage += cap.damage_groups[group]
-            * limit(actual_interval / cap.full_punch_interval, 0.0, 1.0)
-            * (object.armor_groups[group] / 100.0)
-            -- Where object.armor_groups[group] is 0 for inexistent values
-    return damage
+```lua
```

Comment:
this looks to be some sort of python-lua hybrid pseudocode. could change it to be more in line with lua to make the syntax highlighting better

Reply:
I'm not really sure how the damage calculation works.

Reply:
Fixed!